src/Microsoft.CodeQuality.Analyzers/Core/Microsoft.CodeQuality.Analyzers.Core.csproj

```diff
@@ -46,7 +46,7 @@
     <EmbeddedResource Update="Documentation\XmlDocumentationCommentsAnalyzersResources.resx">
       <Generator>ResXFileCodeGenerator</Generator>
       <LastGenOutput>XmlDocumentationCommentsAnalyzersResources.Designer.cs</LastGenOutput>
-      <CustomToolNamespace>Documentation\XmlDocumentationComments.Analyzers</CustomToolNamespace>
```

Comment:
I verified that regenerating the .designer.cs on my machine produces no diffs, so the custom tool was already stripping off characters before the last `\`